flw/src/main/resources/META-INF/spring/blueprint.xml

```diff
@@ -33,6 +33,9 @@
     <osgi:service ref="whitelistService"
                   interface="org.motechproject.nms.flw.service.WhitelistService" />
 
+    <osgi:service ref="itService"
```

Comment:
nothing to do now but should we consider adding an ops task to 'unwire' the itService dependency for production?